include/tlapack/lapack/potri.hpp

```diff
@@ -0,0 +1,88 @@
+/// @file potrf2.hpp Computes the Cholesky factorization of a Hermitian positive
```

Comment:
potri.hpp, not potrf2.hpp

Reply:
And well change the comment, you can look at LAPACK dpotri.f to take inspiration.